### PR TITLE
✨ Query if iscn_id mapping exists for liker + url

### PR DESCRIPTION
**Pull request by williamchong**

### CODE REVIEW

File: src/util/api/api.js [hunk 1]
```diff
@@ -17,6 +17,8 @@ function getLikeCoinButtonHeaders({
 
 export const apiGetUserMinById = id => axios.get(`${LIKECOIN_API}/users/id/${id}/min`);
 
+export const apiGetURLToISCNMapping = (likerId, referrer) => axios.get(`${LIKECOIN_API}/like/iscn/mapping?liker=${encodeURIComponent(likerId)}&url=${encodeURIComponent(referrer)}`);
```
Comment (Tsai-Hsueh-Kuan) [inline src/util/api/api.js:20]: https://github.com/likecoin/likecoin-api-private/pull/352#:~:text=Also%20liker%20should,1
need change to likerId

Comment (williamchong) [reply src/util/api/api.js:20]: for qs please do underscore, i.e. `liker_id`

Comment (Tsai-Hsueh-Kuan) [reply src/util/api/api.js:20]: ok, fix to liker_id ok in private api

Comment (Tsai-Hsueh-Kuan) [reply src/util/api/api.js:20]: help me change it to liker_id here, thx

**Pull request merged**